krabcake/rs_hello/src/lib.rs

```diff
@@ -558,15 +580,18 @@ unsafe fn check_use_1(addr: vg_addr, shadow_addr: vg_ulong) {
     // 3. as a side-effect of the access, we must pop all entries that
     //    lay above the aforementioned Unique(T).
     //TODO(bryangarza): Have to bring this fn back from the dead???
+    let tag = Tag::from_vg_ulong(shadow_addr).unwrap();
```

Comment:
this code is within a method named `check_use_1`, which is meant to be read as "check (the rule) use-1".

Given that you are updating it to correspond to  the rule `use-2` in the Stacked Borrows paper, lets rename this method to either "check_use_2", or alternatively, just call it "check_use" (because the use-1 vs use-2 distinction in the paper is just a presentation detail for how they gradually introduce the overall system).